.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    commit-message:
+      prefix: "ci(dependabot):"
+    labels:
+      - "maintenance"
```

This needs to be updated with our label name I think
```suggestion
      - ":wrench: type: Maintenance"
```